GNUmakefile.main

```diff
@@ -0,0 +1,113 @@
+BRANCH := main
+DBTYPE := boltdb
+DBPATH := ~/.cache/vuls/vuls.db
+
+.PHONY: db-build
+db-build:
+	vuls db init --dbtype ${DBTYPE} --dbpath ${DBPATH}
+	make db-add REPO=vuls-data-extracted-alma-errata BRANCH=${BRANCH} DBTYPE=${DBTYPE} DBPATH=${DBPATH}
```

Comment:
No significant difference in this PR, would it be better to use $(MAKE) to invoke child make ?
cf. https://stackoverflow.com/a/68240460

Reply:
fixed: https://github.com/vulsio/vuls-data-db/compare/d25af38cc2ed1d5dbe77f9ec5d54c2bae8d2cc94..1a8a8b5e9555517b762e1f66ea94d4ba88015af7